pycbc/io/live.py

```diff
@@ -59,32 +63,37 @@ def __init__(self, ifos, coinc_results, **kwargs):
         self.template_id = coinc_results[f'foreground/{ifos[0]}/template_id']
         self.coinc_results = coinc_results
         self.ifos = ifos
+        assert len(coinc_ifos) < 3, f"I can't handle {coinc_ifos} coinc ifos!"
+        self.psds = kwargs['psds']
```

Comment:
Whan happens with singles then?

Reply:
I expect coinc_ifos to be length 1 .. ??

Reply:
Actually I meant to remove this assert and only have the one for the pastro calculation, since there is no fundamental reason to exclude 3 coinc ifos here